src/ripple/app/paths/impl/FlowDebugInfo.h

```diff
@@ -161,6 +161,7 @@ struct FlowDebugInfo
                 auto const end = FlowDebugInfo::clock::now ();
                 info->timePoints[tag].second = end;
             }
+            Stopper(Stopper&&) = default;
```

Comment:
Right on all counts.  In this case the compiler complained about the use of the implicit/deprecated copy constructor on the return two lines down from here.  On noting that this was the only place the copy constructor was used, and that it had a `string` member, I decided to give it a move constructor instead of a copy constructor.

In C++17 we will be able to remove the move constructor and delete the copy members of `Stopper`.